plugins/export.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: default
+title: Export plugin
+title_nav: Export
+description: Export content from TinyMCE, into various formats.
+keywords: plugin export pdf
+---
+
+{% assign pluginname = "Export" %}
+{% assign plugincode = "export" %}
+
+{% assign beta_feature = "The Export plugin" %}
+{% include misc/beta-note.md %}
+
+{{site.requires_5_5v}}<br/>
+{{site.premiumplugin}}
+
+The {{pluginname}} plugin adds the ability to export content from the editor to a user's local machine in various formats. For a list of available exporters and information on what they support, see the [Exporters section](#exporters).
+
+## Interactive example
+
+To export the editor content, either:
+
+- From the _File_ menu, click _Export_ and select _PDF_.
+- Click the Export toolbar button (![Export icon: A page with an arrow from the center of the page to the right of the page]({{site.baseurl}}/images/icons/export.svg)) and select _PDF_.
+
+{% include live-demo.html id="export" %}
+
+## Basic setup
+
+To add the {{pluginname}} plugin to the editor, add `{{plugincode}}` to the `plugins` option in the editor configuration.
+
+For example:
+
+```js
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: '{{plugincode}}',
+  toolbar: 'export',
+  export_image_proxy: 'proxy.php' // Required for rendering remote images
+});
+```
+
+## Exporters
+
+The {{pluginname}} plugin provides the following exporters:
+
+- [Client-side PDF (`clientpdf`)](#client-side-pdf)
+
+### Client-side PDF (`clientpdf`)
+
+The client-side PDF exporter converts the editor content to a PDF without the need for server-side components. This exporter will resize the content to fit on A4 pages, add page breaks, take a snapshot of the HTML, and embed the snapshot image within the exported PDF.
+
+This exporter has a few limitations or known issues that should be noted:
+
+- The text content in the PDF cannot be selected or copied.
+- A single line of content sliced horizontally and distributed across separate pages.
+- Due to browser limitations, there is a limit on the number of pages that be rendered. The number of pages varies between browsers.
```

Comment:
The word "limitations" is used a few times in this section, but this sentence in particular feels a bit strange with "limit" being used immediately after.

"Limitations" is mentioned above, I think this could just be

```suggestion
- There is a limit on the number of pages that be rendered. The number of pages varies between browsers.
```

Reply:
We want to call out this specifically as a browser limitation, rather than a limitation of the plugin (ie not something we can resolve).

Reply:
grammar "...  pages **that be rendered**"

Reply:
:facepalm: has been fixed in https://github.com/tinymce/tinymce-docs/pull/1712/commits/6ccc95947c299727854394bc7e4d2d3e84bd41fb